src/dodal/devices/util/epics_util.py

```diff
@@ -1,10 +1,13 @@
 from functools import partial
 from typing import Callable
 
+from bluesky.protocols import Movable
```

Comment:
Nit: I would like it better if we did something like

```python
from ophyd_async.core import device as OphydAsyncDevice
from ophyd import device as OphydDevice
```

or alternatively to use `module.object` syntax for both `ophyd` and `ophyd_async` items